lms/templates/registration/account_activation_sidebar_notice.html

```diff
@@ -0,0 +1,34 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="profile-sidebar" role="region" aria-label="Account Activation Info">
```

Comment:
This aria-label is not translatable.  It would be better to use aria-labelledby and reference the H2 element below (by ID) since that is translated.